### ISSUE TITLE
Unrecognized color

### ISSUE THREAD

**Issue by jwhazel** [martinheidegger/mapslice#20]
Getting this error no matter whether using imagemagick or graphicsmagick.

    Error: Command failed: gm convert: Unrecognized color (0000000).

I fixed it by setting the background color to `rgb(0,0,0)` instead and everything works. Node 6.3.1, both graphics libraries installed via brew if it matters.

**Comment by martinheidegger** [comment]
@jwhazel The documentation mentions to use a hash in front like `#0000000`. Did you try it like that? How did you call mapslice?

**Comment by jwhazel** [comment]
Try running the example script you have posted verbatim, no other code. I'm attaching the code + error output from my console. It seems like something somewhere is stripping the hash out. Not sure why rgb() works though. Apologies in advance if this is on my end, but I can't think of anything exotic in my setup that would cause this.

```
administrators-MacBook-Pro:mapslicer jhazel$ cat index.js
var mapslice = require("mapslice");
 

// The parameters passed here are equal to the command-line parameters
var mapSlicer = mapslice({
    file: "myImage.jpg",               // (required) Huge image to slice
    output: "myImage/{z}/{y}/{x}.png", // Output file pattern
    tileSize: 512,                     // (default: 256) Tile-size to be used
    imageMagick: true,                 // (default: false) If (true) then use ImageMagick instead of GraphicsMagick
    background: "#0000000",            // (default: '#FFFFFFFF') Background color to be used for the tiles. More: http://ow.ly/rsluD
    tmp: "./temp",                     // (default: '.tmp') Temporary directory to be used to store helper files
    parallelLimit: 3,                  // (default: 5) Maximum parallel tasks to be run at the same time (warning: processes can consume a lot of memory!)
    minWidth: 200,                     // See explanation about Size detection below
    skipEmptyTiles: true,              // Skips all the empty tiles
    bitdepth: 8,                       // (optional) See http://aheckmann.github.io/gm/docs.html#dither
    dither: true,                      // (optional) See http://aheckmann.github.io/gm/docs.html#bitdepth
    colors: 128,                       // (optional) See http://aheckmann.github.io/gm/docs.html#colors
});
 
mapSlicer.on("start", function(files, options) {
    console.info("Starting to process "+files+" files.");
});
 
mapSlicer.on("error", function(err) {
    console.error(err);
});
 
mapSlicer.on("progress", function(progress, total, current, file) {
    console.info("Progress: "+Math.round(progress*100)+"%");
});
 
mapSlicer.on("end", function() {
    console.info("Finished processing slices.");
});

mapSlicer.start();

administrators-MacBook-Pro:mapslicer jhazel$ node index.js
Starting to process 41 files.
Progress: 0%
Progress: 2%
{ Error: Command failed: convert: unrecognized color `0000000' @ warning/color.c/QueryMagickColorCompliance/2670.
convert: unrecognized dither method `-colors' @ error/convert.c/ConvertImageCommand/1376.

    at ChildProcess.proc.on.onExit (/Users/jhazel/Documents/htdocs/test/mapslicer/node_modules/gm/lib/command.js:301:17)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5) code: 1, signal: null }
Finished processing slices.
administrators-MacBook-Pro:mapslicer jhazel$ 
```

**Comment by martinheidegger** [comment]
Sorry for not having come at you. I am sure by now the issue is irrelevant to you but thank you for reporting it still, as maybe someone else might profit from it.

This was an issue in the documentation (there was a "0" missing). I updated the documentation. The fix is also published to npm as 2.0.3.
